### PR TITLE
Create helm chart for scf

### PR DESCRIPTION
**Pull request by jandubois**
Adds `make helm` target to create the chart and adapt `make run` to deploy it.

Changing the DEFAULT_STACK to opensuse42 has been approved by Troy.

This PR needs the following PR's to be approved first:

https://github.com/SUSE/uaa-fissile-release/pull/27
https://github.com/SUSE/fissile/pull/252
https://github.com/SUSE/fissile/pull/257

After the fissile PR's have been approved, this PR needs to be amended to update the URL of the fissile binary in `bin/dev/install_tools.sh`.

Or for testing the new binary can be copied manually after `vagrant up` with something like: `cp build/linux-amd64/fissile ~/hpcloud/hcf/output/bin/`.

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for SUSE Cloud Foundry
+name: cf
```
Comment (viovanov) [inline Chart.yaml]: `scf`?

Comment (viovanov) [inline Chart.yaml]: There's more metadata we can add here.
Can you ping @troytop to give us some values?

Comment (jandubois) [inline Chart.yaml]: I've pinged Troy, he is currently busy with other stuff.

I've created a separate ticket for it: https://trello.com/c/6E77Oadd/213-provide-metadata-for-the-chartyaml-files-for-scf-and-uaa

File: make/kube [hunk 2]
```diff
@@ -3,19 +3,27 @@
...
+# TODO: We currently skip memory limits for development purposes
+fissile build ${BUILD_TARGET} \
+    --output-dir=${BUILD_TARGET} \
     --defaults-file="$(echo "${ENV_FILES}" | tr ' ' ,)" \
-    --use-memory-limits=false \
+    --use-memory-limits=false
 
 # This is a small hack to make the output of make kube be compatible with K8s 1.6
-perl -p -i -e 's@extensions/v1beta1@batch/v1@' kube/bosh-task/*.yml
+perl -p -i -e 's@extensions/v1beta1@batch/v1@' $(grep -rl "kind: Job" ${BUILD_TARGET})
```
Comment (mook-as) [inline make/kube]: Quotes around `${BUILD_TARGET}` (yes I know there are no spaces; it's a shellcheck thing)

File: make/run [hunk 3]
```diff
@@ -32,10 +32,19 @@ kubectl get storageclass persistent 2>/dev/null || {
...
 }
 
-kubectl create --namespace="${NAMESPACE}" \
-    --filename="kube/bosh" \
-    --filename="kube/secrets" \
-    --filename="kube/bosh-task/post-deployment-setup.yml"
+source bin/settings/settings.env
+source bin/settings/network.env
+namespace=${NAMESPACE}
+source bin/settings/kube/hcp-compatibility.env
```
Comment (mook-as) [inline make/run]: Do we still have things in there that are useful? Huh. We should consider moving them to the main settings env (possibly in a different PR)

Comment (jandubois) [inline make/run]: It only has `HCP_SERVICE_DOMAIN_SUFFIX`. I thought that was necessary, but maybe I was confused.

**Comment by mook-as** [comment]
Merging; smoke tests passed locally

**Pull request merged**